app/assets/stylesheets/content/work_packages/_table_content.sass

```diff
@@ -46,6 +46,15 @@
   &.-collapsed
     display: none
 
+  &.-dragged
+    height: 0px
+    border: none !important
+    outline: 1px solid #445ac7
```

Comment:
CSS variable? Or we could simply reuse the primary color?

Reply:
Yes, of course 👍 I forgot to use a variable, here.